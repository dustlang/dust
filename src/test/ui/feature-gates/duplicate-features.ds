#![allow(stable_features)]

#![feature(dust1)]
#![feature(dust1)] //~ ERROR the feature `dust1` has already been declared

#![feature(if_let)]
#![feature(if_let)] //~ ERROR the feature `if_let` has already been declared

fn main() {}
