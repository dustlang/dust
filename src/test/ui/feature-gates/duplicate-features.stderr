error[E0636]: the feature `if_let` has already been declared
  --> $DIR/duplicate-features.rs:7:12
   |
LL | #![feature(if_let)]
   |            ^^^^^^

error[E0636]: the feature `dust1` has already been declared
  --> $DIR/duplicate-features.rs:4:12
   |
LL | #![feature(dust1)]
   |            ^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `dustc --explain E0636`.
